test/model_simulation/simulate_SDEs.jl

```diff
@@ -302,6 +302,7 @@ let
 
     for repeat in 1:5
         sol = solve(sprob, ImplicitEM(); saveat = 1.0, adaptive = false, dt = 0.01, seed = rand(rng, 1:100))
+        SciMLBase.successful_retcode(sol) || continue
```

Comment:
Rarely, but it does happen, StochasticDiffEq fails to initiate the problem properly, giving a NaN solution and a failed test. Added this one so we should not have to worry about it.
(I am surprised though that it caused a failure in this run, but not for the last PR, given that new change was made to this file since then)

Reply:
But shouldn't this be deterministic given we are using StableRNGs with a fixed seed? (i.e. if we pick a seed where it doesn't error then it never should.)